File: dotnetcore/StarterBot/Entities/PowerUp.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using StarterBot.Enums;

namespace StarterBot.Entities
{
    public class PowerUp
    {
        [JsonConverter(typeof(StringEnumConverter))]
        public PowerUpType Type { get; set; }
        public int Value { get; set; }
    }
}

File: java/src/main/java/za/co/entelect/challenge/entities/PowerUp.java
package za.co.entelect.challenge.entities;

import com.google.gson.annotations.SerializedName;
import za.co.entelect.challenge.enums.PowerUpType;

public class PowerUp {
    @SerializedName("type")
    public PowerUpType type;

    @SerializedName("value")
    public int value;
}
